gatsby-node.js

```diff
@@ -0,0 +1,47 @@
+const path = require(`path`)
+const { createFilePath } = require(`gatsby-source-filesystem`)
+
+exports.onCreateNode = ({ node, getNode, actions }) => {
+    const { createNodeField } = actions
```

Comment:
It looks like you have some inconsistent spacing in different JS files here. There's...a lot of debate about the right way to indent things, but a lot of JS libraries and some other major tools we use have standardized on 2 spaces. I'd suggest 

- Picking a spacing you like (4 spaces, 2 spaces, or tab characters) and setting your editor to always use that value
- Reviewing these files and making sure they're all indented the same way (most editors have a "re-indent" command you can use to automatically transform these files)
- If you get eslint set up, make sure to configure your spacing — after that, eslint will yell at you whenever your spacing is inconsistent